src/Playwright/Core/Page.cs

```diff
@@ -1357,6 +1357,9 @@ await Context.RecordIntoHarAsync(har, this, new()
                 Url = options?.Url,
                 UrlString = options?.UrlString,
                 UrlRegex = options?.UrlRegex,
+                Update = options.Update,
+                UpdateContent = options.UpdateContent,
```

Comment:
```suggestion
                UpdateContent = options?.UpdateContent,
```
here and below?

Reply:
I fell for it as well but above is already a `options?.Update == true` check so `options` can't be `null` here.
Still might be good to make it consistent.

Reply:
Agreed. @mxschmitt let's make it consistent.

Reply:
good point! addressed.